public/index.html

```diff
@@ -25,7 +25,7 @@
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
+    <div id="root" class="root-container"></div>
```

Comment:
Please remove this commit

Reply:
@nvminhtue thanks, I have rebased again to remove this commit